File: Python/1048.py
valor = float(input())
ajuste = 0.00
if (valor <= 400):
	ajuste = 0.15
elif (valor <= 800):
	ajuste = 0.12
elif (valor <= 1200):
	ajuste = 0.10
elif (valor <= 2000):
	ajuste = 0.07
else:
	ajuste = 0.04

ganho = ajuste * valor
novo_valor = valor + ganho

print (f"Novo salario: {novo_valor:.2f}")
print (f"Reajuste ganho: {ganho:.2f}")
print (f"Em percentual: {(ajuste*100):.0f} %")

File: Ruby/1048.rb
valor = gets.chomp.to_f
ajuste = 0

if (valor <= 400) then
	ajuste = 0.15
elsif (valor <= 800) then
	ajuste = 0.12
elsif (valor <= 1200) then
	ajuste = 0.10
elsif (valor <= 2000) then
	ajuste = 0.07
else
	ajuste = 0.04
end

ganho = ajuste * valor
novo_valor = valor + ganho

puts ("Novo salario: %0.2f" % novo_valor)
puts ("Reajuste ganho: %0.2f" % ganho)
puts ("Em percentual: %d %%" % (ajuste * 100))
